### DISCUSSION TITLE
after a while NotifyClose receive <nil>

### DISCUSSION THREAD

**Discussion by parsibox** [rabbitmq/amqp091-go#264] [Q&A] [answered]
hi
i use  Version github.com/rabbitmq/amqp091-go v1.8.1 with go 1.21  and RabbitMQ 3.11.16
after some hours in NotifyClose   i get <nil>  and i can not get more ditail
what is it problem?
is it means my connection disconnected?
what should i do when receive <nil> on NotifyClose  ?should i close current conenction and reconnect?

**Comment by lukebakken** [comment, marked as answer]
Hello, thanks for using RabbitMQ and this library.

Please note that RabbitMQ 3.11.x is completely out of support ([support policy](https://github.com/rabbitmq/rabbitmq-server/blob/main/COMMUNITY_SUPPORT.md)).

To receive community support, you must use the latest version of RabbitMQ and this library.

You should also see what RabbitMQ logs at the same time `NotifyClose` is called. That will probably tell you what is going on.
